code-review/README.md

```diff
@@ -14,20 +14,14 @@ For more information, review the [ISMS policy documentation in Aptible][isms-pol
 
 ### Policy
 
-* For every Healthify codebase that contains sensitive data, one or many code
+* For every Healthify codebase that contains sensitive code, one or many code
   maintainers are defined.
-* A code maintainer (CM) is defined as someone that has worked at Healthify for 90
-  days and has been approved by engineering leadership to be a code maintainer.
-* If you are a CM:
-  * When developing software you still must receive code review, but code review
-    may happen by any developer, whether they are a CM or not.
-* If you are not a CM:
-  * When developing software, you must receive an approved code review by a CM.
-  * When performing code review, you must not approve a pull request in the
-    Github web UI.
-    * With that said, you can (and are encouraged to) perform code review.
-    * After completing your review, request review from a CM so that they can
-      sign-off on the pull request.
+* Every pull request (PR) to such a codebase requires an explicit Github 'Approve' action from
+a qualified code reviewer. If the code has been paired on, then one of the members of the pair can 'Approve' it.
+* A code maintainer (CM) is defined as someone that has has been approved by engineering leadership to be a code maintainer, typically after working at Healthify for 90 days. A code maintainer may give an 'Approve' to any PR.
+* When you first join Healthify, you may not 'Approve' any PR, but you may comment or review them and in fact you are encouraged to!
+* After 30 days at Healthify and a sync up with your manager, you may be granted privileges to 'Approve'
```

Comment:
@KrishnaKulkarni I'm not sure I understand this last guideline. It seems to conflict with the idea of a 90 maintainer. The way this reads to me is:
- At 90 days, you become a CM. Before that, you cannot approve any PR.
- But at 30 days, you CAN approve a PR, if it was submitted by a CM.

Given how infrequently new devs are on the team (ie, under 90 days), this seems a little confusing. I can see where this can happen, but maybe it can be reworded, or at least the bullet points re-ordered, like (1) 0 day mark, (2) 30 day mark, (3) 90 day mark

Reply:
Agree with this language confusion. I would recommend adding the language as a sub-bullet for explicitness:

" * However, you can *not* approve a PR submitted by a non-CM"